bot/commands/usr_kaamo.py

```diff
@@ -268,4 +268,4 @@ async def cmd_kaamo(message : discord.Message, args : str, isDM : bool):
                     longHelp="List all items in your Kaamo Club storage. Kaamo has a max capacity of " \
                         + str(cfg.kaamoMaxCapacity) + " items, including items on ships. Give an item type " \
                         + "(ship/weapon/turret/module/tool) to only list items of that type.\n\n" \
-                        + "⚠ Please be aware that `kaamo store` and `kaamo get` can only be used be level 10 bounty hunters.")
+                        + "⚠ Please be aware that `$kaamo store` and `$kaamo get` can only be used by level 10 bounty hunters.")
```

Comment:
Thanks so much for the fix!

I've omitted `$` from help strings because it might not always apply - each guild can set their own command prefix with `set-prefix`.

It does still make some sense though, since `$` is the default command prefix - we could change it to `cfg.defaultCommandPrefix`?